servers/src/main/java/tachyon/worker/netty/PipelineHandler.java

```diff
@@ -39,8 +42,9 @@ public PipelineHandler(final BlocksLocker locker, final TachyonConf tachyonConf)
   protected void initChannel(SocketChannel ch) throws Exception {
     ChannelPipeline pipeline = ch.pipeline();
     pipeline.addLast("nioChunkedWriter", new ChunkedWriteHandler());
-    pipeline.addLast("blockRequestDecoder", new BlockRequest.Decoder());
-    pipeline.addLast("blockResponseEncoder", new BlockResponse.Encoder(mTachyonConf));
-    pipeline.addLast("dataServerHandler", new DataServerHandler(mLocker));
+    pipeline.addLast("frameDecoder", RPCMessage.createFrameDecoder());
+    pipeline.addLast("RPCMessageDecoder", new RPCMessageDecoder());
+    pipeline.addLast("RPCMessageEncoder", new RPCMessageEncoder());
```

Comment:
I personally think RPCMessage* is nicer than rpcMessage. But, this is just a name so I don't think it matters too much.